internal/promotion/runner/builtin/schemas/yaml-merge-config.json

```diff
@@ -0,0 +1,28 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "title": "YAMLMergeConfig",
+
+  "type": "object",
+  "required": ["inPaths", "outPath"],
+  "additionalProperties": false,
+  "properties": {
+    "inPaths": {
+      "type": "array",
+      "description": "InPaths is the list of paths of YAML files to merge",
+      "minItems": 1,
+      "items": {
+        "type": "string"
+      }
+    },
+    "outPath": {
+      "type": "string",
+      "description": "OutPath is the path to the merged YAML file to created or updated.",
+      "minLength": 1
+    },
+    "ignoreMissingFiles": {
+      "type": "boolean",
+      "description": "allow directive to pass even if an input file does not exist.",
+      "default": false
+    }
```

Comment:
Just want to say thank you for this. I like this a lot better that "strict" and being false so it's opt-in feels correct. Really appreciate it.

Reply:
it feels better to me too